public_suffix_list.dat

```diff
@@ -12060,19 +12060,24 @@ ravendb.run
 
 // HOSTBIP REGISTRY : https://www.hostbip.com/
 // Submitted by Atanunu Igbunuroghene <publicsuffixlist@hostbip.com>
-bpl.biz
-orx.biz
-ng.city
+org.cologne
 biz.gl
 ng.ink
```

Comment:
A random spot-check showed that this domain is not currently registered.  Perhaps it has expired, but regardless, section administrators are responsible to update things to remove stale entries.

Please revise this entire section to remove any names that are no longer registered